CHARTER.md

```diff
@@ -0,0 +1,71 @@
+
+TF is leveraging the existing editing [GitHub](https://github.com/w3c/editing) repo with well-known locations and history of discussions on a variety of editing-related topics in order to “incubate” new editing-specific standards.
+
+This charter is intended to reflect on the current direction of the TF group, so that there is common agreement. It may be altered at any point in order to reflect new priorities or work items. 
+  
+
+### 1. Scope
+
+The scope of Editing TF covers multiple aspects of editing, which may  include:
+
+- Textual input and text manipulation 
+- Text editing related events 
+- Selection 
+- Clipboard 
+- Spellcheck and grammar checking 
+- Reusing or discarding parts of or entire pre-existing editing APIs to serve as primitives of editing systems 
+- Standardization of under-specified existing editing related features 
+- Highlighting parts of text ranges without inducing DOM mutations. 
+
+Experts who are contributing members of the task force will contribute to documents produced by the group, as described in the section on deliverables below. 
+ 
+
+### 2. Deliverables
+
+This Task Force will
+- ensure that browsers are developing APIs that meet the expectations of editing framework authors of varying skill and team sizes.   
+- allow a network of experts to share information about gaps and requirements for building editing frameworks on the web.  
+
+The specifications the Editing Taskforce is producing may be driven to the spec level directly by its members within the Web Applications Working Group, or be transferred to another, more appropriate working group after initial collection of requirements. Furthermore, the group defines its success as when it at the very least, produces a recommendation with well defined requirements that result in enhancing existing text editing related specification drafts or helps design new ones.
+
+The TF is expected to work on the following efforts until the end of the charter but it is not meant to be an exhaustive list:
+
+- [Async Clipboard API](https://bugs.chromium.org/p/chromium/issues/detail?id=931839) 
+- [ContentEditableDisabled](http://w3c.github.io/editing/contentEditableDisabled.html) - ability to disable system UI 
+- [EditContext API](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/master/EditContext/explainer.md) 
+- [Highlight API](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/master/highlight/explainer.md) 
+- Native Selection and Caret behaviors 
+- [SpellChecker API](https://github.com/w3c/editing/issues/166) 
+- [SIP Policy](https://github.com/whatwg/html/issues/4876) 
+- [Input Events](https://www.w3.org/TR/input-events-1/) 
+- [ContentEditable](https://w3c.github.io/contentEditable/) 
+
+Deliverables are expected to eventually be shipped as standards recommendation by the Web Applications Working Group, CSS Working Group or any other group where a draft specification may fall into.
```

Comment:
@ylafon or @siusin this seems odd with regards to deliverables... is it normal to a group to operate across working groups?

Reply:
@marcoscaceres In practice this is what we do. An example is the Highlight API which we sent on to the CSSWG rather than the Web Apps WG.

Reply:
I guess eventually this will be a multi-group deliverable? Won't the CSS WG also ask an an incubation process?

Reply:
The CSS WG has decided to take it on, as an ED at first, to be promoted to a FPWD when sufficiently mature (in other words, it has decided to incubate it). Input form the Editing TF is more than welcome, but I don't think we need to consider it a multi-WG deliverable.

Saying that the deliverables are expected to go on the REC track within the Web Apps WG seems right. We should probably also add a liaison section, mention other WGs (such as CSS, or WHATWG…) with which we expect to discuss topics of common interest, and possibly (though not necessarily) list specific such topics, such as the Highlight API.

Reply:
> The CSS WG has decided to take it on, as an ED at first, to be promoted to a FPWD when sufficiently mature (in other words, it has decided to incubate it). Input form the Editing TF is more than welcome, but I don't think we need to consider it a multi-WG deliverable.

oh, that's great news. 

> Saying that the deliverables are expected to go on the REC track within the Web Apps WG seems right. We should probably also add a liaison section, mention other WGs (such as CSS, or WHATWG…) with which we expect to discuss topics of common interest, and possibly (though not necessarily) list specific such topics, such as the Highlight API.

Agree. That would also align really nicely with the [discussion we had yesterday with the HTML Editors](https://github.com/w3c/contentEditable/issues/2) about `contentEditable`.